components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -80,9 +82,15 @@ export const PrebuildDetailPage: FC = () => {
             }
         });
         logEmitter.on("logs-error", (err: ApplicationError) => {
+            if (err.code === ErrorCodes.NOT_FOUND) {
+                setLogNotFound(true);
+                return;
+            }
+
             toast("Fetching logs failed: " + err.message, { autoHide: false });
         });
-    }, [logEmitter, toast]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [prebuild, toast]);
```

Comment:
I'm not sure why we change it, but not a blocker if it just works

Reply:
Thanks for catching this. It's just an oversight from the previous changes